core/backend/src/BriefcaseManager.ts

```diff
@@ -318,6 +318,7 @@ export class BriefcaseManager {
       await IModelHost.iModelClient.briefcases.get(requestContext, iModelId, new BriefcaseQuery().byId(briefcaseId));
       requestContext.enter();
     } catch (err) {
+      Logger.logInfo(loggerCategory, "Could not find briefcase to release", () => ({ iModelId, briefcaseId }));

```

Comment:
@paulius-valiunas i would just take out the try/catch and let callers deal with this. How did they get the invalid ids to begin with?

Reply:
I didn't get invalid ids, but there were cases where the tile agent didn't release its briefcase for unknown reasons. I got the log message from line 332 and it was totally useless.

I wanted to just remove the try/catches in this method, but I thought someone might have a reason to have it this way. I know you rewrote a big part of this class recently, so if you think it's fine to just throw these errors, I'll be happy to remove them.

Reply:
I didn't change anything in this code, but i don't think it's right for it to swallow errors. Nobody can be relying on that.

Reply:
yeah that makes sense, I just wanted to hear a second opinion. I changed it to throw these errors but kept one of the catch blocks to add that log message. That should make it easier to see which of the two requests failed.